include/mbedtls/md.h

```diff
@@ -38,6 +38,8 @@
 #define MBEDTLS_ERR_MD_ALLOC_FAILED                       -0x5180  /**< Failed to allocate memory. */
 #define MBEDTLS_ERR_MD_FILE_IO_ERROR                      -0x5200  /**< Opening or reading of file failed. */
 
+// TODO [TR] for #4029: can't remove it because it's still used in the code.
+// see the other TODOs
```

Comment:
Note: we usually avoid leaving TODO comments in the source code. In this instance it's OK since we already have a github issue tracking that, you'll be working on it soom, but in the future please use github issue rather than in-source comments for tracking TODOs.

Reply:
That was my idea while adding the TODO (to remove it later).